File: shared/src/main/java/org/dwallach/calwatch/WireEvent.java
/*
 * CalWatch
 * Copyright (C) 2014 by Dan Wallach
 * Home page: http://www.cs.rice.edu/~dwallach/calwatch/
 * Licensing: http://www.cs.rice.edu/~dwallach/calwatch/licensing.html
 */
package org.dwallach.calwatch;

/**
 * Created by dwallach on 12/18/14.
 */
public final class WireEvent {
    public final Long startTime;
    public final Long endTime;
    public final Integer displayColor;

    public WireEvent(Long startTime, Long endTime, Integer displayColor) {
        this.startTime = startTime;
        this.endTime = endTime;
        this.displayColor = displayColor;
    }
}

File: app/src/main/kotlin/org/dwallach/calwatch/WireEvent.kt
/*
 * CalWatch
 * Copyright (C) 2014 by Dan Wallach
 * Home page: http://www.cs.rice.edu/~dwallach/calwatch/
 * Licensing: http://www.cs.rice.edu/~dwallach/calwatch/licensing.html
 */
package org.dwallach.calwatch

/**
 * This data structure handles a given calendar event. At one point in history, we were sending
 * these things from the phone to the watch, but that's no longer necessary as it's available
 * locally.
 */
data class WireEvent(val startTime: Long, val endTime: Long, val displayColor: Int)
